src/frontend/templates/pages/Login.js

```diff
@@ -70,6 +74,10 @@ export const Login = () => {
                         <button type='submit' className='btn btn-calm d-flex jc-c w-100 f-w-600 bdr-rad-md'>Login</button>  
                         </div>
                         </form>
+                        <div className='d-flex px-lg'>
+                        <button onClick={(e) => testLogin(e)} className='btn btn-eco d-flex jc-c w-100 f-w-600 bdr-rad-md'>Enter Test Credentials</button>  
```

Comment:
Any particular reason why event is passed to the testLogin function?

Reply:
hi tejas , it doesn't serves any purpose will remove it ! thanks for pointing that out